Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM elixir:1.5.2-alpine
+
+RUN apk update && apk add bash
```

Comment:
Why you install `bash`?

Reply:
Because if I don't I get this message when running my release: `Bash is required to run this release. Please ensure that it is installed.`